src/nvim/lua/vim.lua

```diff
@@ -279,7 +279,19 @@ setmetatable(vim, {
 
 do
   local a = vim.api
+  local nvim_command = a.nvim_command
   local validate = vim.validate
+  vim.cmd = setmetatable({}, {
+    __index = function(_, k)
+      return function(s)
+        validate { cmd = {s, 's', true} }
```

Comment:
style nit: when calling "kwargs" function it makes sense to omit space after the function name, just as with parens.

```
foo(...)
foo{...}
```

Reply:
I'm not sure why I do that to be honest. I think my eyes just see it better if its separate.

Reply:
#11312